java-client/src/main/java/org/opensearch/client/opensearch/cluster/AllocationExplainRequest.java

```diff
@@ -37,12 +37,7 @@
 import java.util.Map;
 import java.util.function.Function;
 import javax.annotation.Nullable;
-import org.opensearch.client.json.JsonpDeserializable;
-import org.opensearch.client.json.JsonpDeserializer;
-import org.opensearch.client.json.JsonpMapper;
-import org.opensearch.client.json.ObjectBuilderDeserializer;
-import org.opensearch.client.json.ObjectDeserializer;
-import org.opensearch.client.json.PlainJsonSerializable;
+import org.opensearch.client.json.*;
```

Comment:
I am surprised this doesn't get trapped by spotless, let's expand these like in all other places (and check spotless configuration please).

Reply:
Yeah that would be my ide auto import :) , I will fix these.